spec/support_specs/basic_auth_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'rails_helper'
```

Comment:
We use the `support_specs` file for testing test machinery like the OpenAPI parser. These seem like request specs, maybe?

Reply:
I put the spec file there because basic auth, they way we are doing it, is essentially a 'cloaking' feature, affecting all parts of the app and potentially one we'll discard when we go live. If I were to move the examples to request/ I'd have to split them across the different interfaces, which would make them harder to find as a single feature.

Reply:
Moved to system/ because it's testing a feature.